contributing/telemetry.md

```diff
@@ -4,9 +4,12 @@ Similar to what vscode does, we are reporting on certain events that happen in t
 
 ## How it works
 
-All extensions leverage the telemetryService in `salesforce-vscode-core` extension in order to send data. The service in `salesforce-vscode-core` is responsible for showing an information message to users and properly initializing the service for other extensions in this repository to leverage. Similar to VSCode, we provide the user an opt out mechanism. We check that user settings `telemetry.enableTelemetry` and `salesforcedx-vscode-core.telemetry.enabled` are enabled before initializing the service.
+All extensions leverage the telemetryService in `salesforce-vscode-core` extension in order to send data. 
```

Comment:
There were some extensions like visualforce and debugger telemetry that did something different. Do these extensions also leverage the core extension telemetry?

Reply:
I just deleted the part I knew to be totally wrong.  AFAIK every extension in the monorepo has it's own telemetry service created via the shared utils package, and external extensions to the monorepo use the core extension to get access to creating a TelemetryService.  Can flush this out more.